packages/diagnostic/src/DiagnosticService.ts

```diff
@@ -0,0 +1,79 @@
+import ts from 'typescript';
```

Comment:
we have been using kebab case for filenames. `DiagnosticService.ts` should be changed to `diagnostic-service.ts`